crates/control/src/fake_data.rs

```diff
@@ -14,16 +14,18 @@ use types::{
 pub struct FakeData {}
 
 #[context]
+#[allow(dead_code)]
```

Comment:
These fields are used to generate the databases and configuration structs for the behavior simulator.
However, this node is currently never instantiated, so all of the fields are unused.
Since I removed the `pub`s, the compiler now knows this.